### PR TITLE
Headless priority class fix

### PR DESCRIPTION
**Pull request by at88mph**
Correctly set the `headlessPriorityGroup` setting to Skaha.

### CODE REVIEW

**Comment by at88mph** [comment]
CADC-13748

File: deployment/helm/README.md [hunk 1]
```diff
@@ -267,10 +267,17 @@ deployment:
     # See https://www.ivoa.net/documents/GMS/20220222/REC-GMS-1.0.html#tth_sEc3.2
     adminsGroup: "ivo://example.org/gms?prototyping-groups/mini-src/platform-users"
 
-    # Group for users to preempt headless jobs.
+    # Group URI for users to preempt headless jobs.
```
Comment (brianmajor) [inline deployment/helm/README.md]:
```suggestion
    # Group URI for that allow members to run headless jobs.
```

File: deployment/helm/README.md [hunk 2]
```diff
@@ -267,10 +267,17 @@ deployment:
...
     # See https://www.ivoa.net/documents/GMS/20220222/REC-GMS-1.0.html#tth_sEc3.2
     headlessGroup: "ivo://example.org/gms?prototyping-groups/mini-src/platform-users"
```
Comment (brianmajor) [inline deployment/helm/README.md]:
```suggestion
    headlessGroup: "ivo://example.org/gms?skaha-headless-users"
```

File: deployment/helm/skaha/sample-local-values.yaml [hunk 3]
```diff
@@ -15,9 +15,15 @@ deployment:
...
     # The group name to verify users against for permission to use the Science Platform.
-    usersGroup: "ivo://example.org/gms?/example/users-group-name"
+    usersGroup: "ivo://example.org/gms?example/users-group-name"
 
-    adminsGroup: "ivo://example.org/gms?/example/admin-group-name"
+    adminsGroup: "ivo://example.org/gms?example/admin-group-name"
+
+    headlessGroup: "ivo://example.org/gms?example/admin-group-name"
+
+    headlessPriorityGroup: "ivo://example.org/gms?example/vip-group-users"
```
Comment (brianmajor) [inline deployment/helm/skaha/sample-local-values.yaml]:
```suggestion
    headlessPriorityGroup: "ivo://example.org/gms?example/headless-priority-group-name"
```

File: deployment/helm/skaha/sample-local-values.yaml [hunk 4]
```diff
@@ -15,9 +15,15 @@ deployment:
     gpuEnabled: false 
 
     # The group name to verify users against for permission to use the Science Platform.
-    usersGroup: "ivo://example.org/gms?/example/users-group-name"
+    usersGroup: "ivo://example.org/gms?example/users-group-name"
 
-    adminsGroup: "ivo://example.org/gms?/example/admin-group-name"
+    adminsGroup: "ivo://example.org/gms?example/admin-group-name"
+
+    headlessGroup: "ivo://example.org/gms?example/admin-group-name"
```
Comment (brianmajor) [inline deployment/helm/skaha/sample-local-values.yaml]:
```suggestion
    headlessGroup: "ivo://example.org/gms?example/headless-group-name"
```

**Review by brianmajor — CHANGES_REQUESTED** [review]
Looks good Dustin.  Just trying to add more consistency to the docs with my comments.  We should remove references to mini-src where they exist.

**Review by brianmajor — APPROVED** [review]

**Pull request merged**